examples/tokens/token_update_transaction_key.py

```
@@ -101,17 +103,20 @@ def update_wipe_key_full_validation(client, token_id, old_wipe_key):
     )
-    
+
     if receipt.status != ResponseCode.SUCCESS:
-        print(f"Token update failed with status: {ResponseCode.get_name(receipt.status)}")
+        print(
+            f"Token update failed with status: {ResponseCode.get_name(receipt.status)}"
+        )
         sys.exit(1)
-    
+
     print(f"Successfully updated wipe key")
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)